src/lib/converter.ts

```diff
@@ -0,0 +1,434 @@
+        item.group,
+      )
+    })
+
+    this.groupedRequest = groups
+      .map((item: CreateRequestGroupType): GroupedRequestType[] =>
+        [...item.paths].map((path) => ({ group: item.group, path })),
+      )
+      .flat()
+    return this.requestGroups
+  }
+
+  /**
+   * @method createRequestMeta
+   * @summary
+   * @param {String} path
+   * @param {String} group
+   * @returns {Object<any>}
+   */
+  private createRequestMeta(path: string, group: string): any {
+    const document = this.apiParser.api as OpenAPIV3.Document
+    const operation = document.paths[path] as OpenAPIV3.PathsObject
+
+    const ctx = Object.entries(operation).map(([verb, value]) => {
+      const requestContext = value as OpenAPIV3.OperationObject
+
+      if (!requestContext || requestContext.deprecated) return null
+
+      const method = verb.toUpperCase()
+      const { summary, description } = requestContext
+      let requestBody = null
+      let responses = null
+      let parameters = null
+      let servers = null
+      let security = null
+
+      if (requestContext.requestBody) {
+        requestBody = requestContext.requestBody
+      }
+
+      if (requestContext.parameters) {
+        parameters = requestContext.parameters
+      }
+
+      if (requestContext.responses) {
+        const accepts = Object.entries(requestContext.responses)
+          .map(([key, value]) => {
+            const ctx = value as OpenAPIV3.ResponseObject
+            return ctx.content ? Object.keys(ctx.content) : []
+          })
+          .flat()
+        responses = accepts.length > 0 ? accepts : '*/*'
+      }
+
+      if (requestContext.servers) {
+        servers = requestContext.servers
+      }
+
+      if (requestContext.security) {
+        security = requestContext.security
+      }
+
+      return {
+        path,
+        group,
+        method,
+        summary,
+        description,
+        requestBody,
+        parameters,
+        responses,
+        servers,
+        security,
+      } as any
+    })
+
+    return ctx
+  }
+
+  /**
+   * @method createRequest
+   * @summary
+   *
+   * @param {Object} item
+   * @param {Number} index
+   * @param {Array<any>} array
+   */
+  private createRequest(item: any, index?: number, array?: any[]): void {
+    const document = this.apiParser.api as OpenAPIV3.Document
+    const { title } = document.info
+
+    const request = this.context.createRequest(
+      item.summary || item.path,
+      item.method,
+      new DynamicString(),
+      item.description,
+    )
+
+    if (item.requestBody) {
+      this.setRequesBody(
+        request,
+        item.requestBody as OpenAPIV3.RequestBodyObject,
+      )
+    }
+
+    if (item.parameters) {
+      this.setRequestParameters(item.parameters, request)
+    }
+
+    if (item.security) {
+      this.setRequestAuth(item.security, request)
+    }
+
+    if (item.responses) {
+      request.setHeader('Accepts', item.responses[0])
```

Comment:
```suggestion
      request.setHeader('Accept', item.responses[0])
```

`Accept` is the right spelling :) 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept